src/utils/helpers/pullErrors.js

```diff
@@ -50,6 +82,30 @@ const pullErrors = (errors) => {
   if (password.length > 0) {
     combinedErrors.password = password;
   }
+  if (gender.length > 0) {
+    combinedErrors.gender = gender;
+  }
+  if (birthday.length > 0) {
+    combinedErrors.birthday = birthday;
+  }
+  if (preferredlanguage.length > 0) {
+    combinedErrors.preferrdlanguage = preferredlanguage;
```

Comment:
what does `preferrdlanguage` mean?. I think it should be `preferredlanguage` as that's more readable.